### PR TITLE
impr(logger): add _log func to control console logging with debug flag

### PR DESCRIPTION
**Pull request by joeldarling**
### Description

Adds `_log` function and `debug` flag, so end-users can turn app logging on or off.

[https://aircall.atlassian.net/browse/ECO-76](https://aircall.atlassian.net/browse/ECO-76)

### Submission

- [x] Your branch is based on `master`
- [x] Your code is unit tested
- [ ] CircleCI builds are green (coverage and lint)
- [x] Add team "CTI" as "Reviewer" for the PR
- [x] Name your PR with the convention `fix|feat|impr(<subject>): description`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ You need to create an instance to use the library. The constructor has a setting
   - `big`: 666px by 376px. Recommanded and default value
   - `small`: 600px by 376px
   - `auto`: 100% width and height. Not recommanded
+- `debug`: Enables or disables logging. Defaults to `true`.
```
Comment (xdurand) [inline README.md]: padding issue?

Comment (joeldarling) [reply README.md]: The lines above are indented as a bullet list :D

File: src/javascripts/aircallPhone.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class AircallPhone {
-  constructor(opts = {}) {
+  constructor(opts = { debug: true }) {
```
Comment (hereismass) [inline src/javascripts/aircallPhone.js]: do we want debug by default?

Comment (joeldarling) [reply src/javascripts/aircallPhone.js]: For now yes :D

**Pull request merged**